Misc/python.man

```diff
@@ -605,20 +681,37 @@ can be set to the callable of your debugger of choice.
 .IP PYTHON_COLORS
 If this variable is set to 1, the interpreter will colorize various kinds of
 output. Setting it to 0 deactivates this behavior.
+.IP PYTHON_BASIC_REPL
```

Comment:
The `PYTHON*` env vars are also (mostly...) in alphabetical order by section in `--help-all`, let's do so here too.